src/main/java/nextstep/optional/User.java

```diff
@@ -25,15 +31,20 @@ public static boolean ageIsInRange1(User user) {
         boolean isInRange = false;
 
         if (user != null && user.getAge() != null
-                && (user.getAge() >= 30
-                && user.getAge() <= 45)) {
+                && (user.getAge() >= MIN_AGE
+                && user.getAge() <= MAX_AGE)) {
             isInRange = true;
         }
         return isInRange;
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(age -> user.age)
```

Comment:
```suggestion
                .map(User::getAge)
```
이렇게 사용해야합니다! 제가 안내를 잘못했네요 ㅠㅠ